tikv/raftstore/applier.go

```diff
@@ -525,33 +528,33 @@ func (a *applier) handleRaftEntryConfChange(aCtx *applyContext, entry *eraftpb.E
 	}
 }
 
-func (a *applier) findCallback(index, term uint64, isConfChange bool) *Callback {
+func (a *applier) findCallback(index, term uint64, isConfChange bool) (*Callback, float64) {
 	regionID := a.region.Id
 	peerID := a.peer.Id
 	if isConfChange {
 		cmd := a.pendingCmds.takeConfChange()
 		if cmd == nil {
-			return nil
+			return nil, 0
 		}
 		if cmd.index == index && cmd.term == term {
-			return cmd.cb
+			return cmd.cb, time.Now().Sub(cmd.startTime).Seconds()
```

Comment:
This can be done in the `Callback.Done`, So we can keep this method signature unchanged.

Reply:
Fixed.